src/controllers/Computer.js

```diff
@@ -0,0 +1,144 @@
+import Controller from "./Controller";
+
+class Computer extends Controller {

```

Comment:
This code requires rework:
1. commented code
2. use of var
3. general clean up